app/utils/check-username.js

```diff
@@ -0,0 +1,27 @@
+import ENV from 'website-my/config/environment'; // remove this when new flow goes live
+
+const BASE_URL = ENV.BASE_API_URL; // remove this when new flow goes live
+
+export default async function checkUserName(userName) {
+  if (!userName) {
+    alert('username field cannot be empty');
```

Comment:
this alert message should come from the constant file

Reply:
what I have realized is that this can be removed as we are already disabling the button if there is nothing written by the user